vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md

```diff
@@ -1,136 +1,101 @@
+## Manage Assets:
 
--   **Download:** transfer the resource to your local machine
+1. Login to the App Builder.
+2. From the left sidebar, go to `Content` → `Assets`.
 
--   **Delete:** remove the resource
+![Digital Asset Main Page](./img/assets-manage.png) 
 
-You can list the contents by filtering with:
+* View `Images` or `Attachments` with the tabs at the top of the page.
+* Search by name or use the `Advanced filters` function to search by `Group`, `Filename` and/or `Category`. 
+* Assets are displayed and ordered by Name. `Choose View` to toggle between a list or grid format. The order can also be changed. 
+* For each resource, click on the action menu (indicated by three vertical dots) to the right to Edit, Delete and preview sizes.
 
--   **All**: All added files are displayed, images and documents both.
+![Digital Asset Information](./img/assets-info.png)
 
--   **Images**: All added images are displayed
+### Add a New Asset
+1. From the Content Assets page in the App Builder, click `Add` on the right to add a new image or document file. 
 
--   **Documents**: All added documents are displayed. 
+![Add Digital Assets Window](./img/assets-add.png)
 
-## Editing Assets
-Once you have a digital asset in place, you have several editing and viewing options. Click the asset's *Actions* button and select **Edit** to update the asset's name/description, categories, and the file represented.
+2. Group: choose from the drop-down menu. This is a required field and defaults to `Free Access`. Additional groups can be defined in the Users section.  
+3. Categories: add your asset to the appropriate category with the `+` symbol under `Join`. This will be used in searching and sorting. This field is not mandatory. New Categories can be created in `Content` → `Categories`.
+4. Drag and Drop your files into the window or click `Browse`. The file you choose will appear in the Name field. You can choose multiple files at one time. You can also use the `Add Another Resource` button and `Choose file` individually.
+5. After all files have been chosen, click the `Add` button to upload the files. This takes you back to the main assets page where your uploaded files will be displayed in the table. If you have an image file, see Edit Assets below. 
+6. If a Name field is left blank, you will receive an error message. Simply go to the actions (indicated by three vertical dots) menu next to the empty file and select Delete. This action menu also provides an edit option for each resource.
 
-### Cropping Images
-You can crop images newly uploaded via Digital Assets. You can either crop the image dynamically or by using pre-configured crop ratios.
+## Edit Assets 
+Changes to the Asset name and categories can be made in the Edit mode.  
+1. From `Content` → `Assets` page, in the list of assets, click on the Asset's `Actions` button and select `Edit`. 
+2. Edit the Name field. For Categories, use the `+` button under `Join` to add the asset to a category. 
```

Comment:
capitalize "asset"? is capitalized in step above

can simplify second sentence in step 2: "Use the `+` button under `Join` to add the Asset to a category." ?